modules/instrument_builder/jsx/react.app.js

```diff
@@ -0,0 +1,485 @@
+TabPane = React.createClass({
```

Comment:
"react.app.js" is a pretty generic name, maybe "instrument_builder.app.js" would be a better name for this file.